modules/custom_operations/user_ie_extensions/tokenizer/CMakeLists.txt

```diff
@@ -66,6 +66,8 @@ target_include_directories(${TARGET_NAME} PRIVATE
   "${CMAKE_BINARY_DIR}/third_party/dart/src/extern_dart/include/"
   "${CMAKE_BINARY_DIR}/third_party/install/re2/include/")
 
+target_include_directories(${TARGET_NAME} PUBLIC .)
```

Comment:
typically, OV extensions does not imply to provide any extra C++ API..

Reply:
It's @slyalin's request to call `pack_strings()` from OV extensions.

Reply:
Please consider the fact that we are going to remove pack_strings/unpack_strigs after string support is added to OV. But we are not really certain when it happens. I think it makes sense to add it now to have functional distribution and single point of reference (people started to copy this function from one place to another because we are not providing distribution for these methods). I hope it won't cross OV releases boundary.